docs/specification.md

```diff
@@ -178,3 +179,10 @@ Takes as input the proposal's IPFS URI and the choice made `x` .
 - fails if the sender already voted on this proposal before
 - fails if less than `vote_delay` has passed since the proposal submission, aka the proposal is still in a `sumbitted` status
 - fails if more than `expire_time` has passed since the proposal submission, aka the proposal has been completed (either as `expired` or `passed`)
+
+### `verify_min_balance`
+
+Callback for the `[balance_of` CPS view of FA2](https://gitlab.com/tezos/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#balance_of), used to check the balance of a `participant` that used the `propose` entrypoint.
```

Comment:
Note that here the first tick should be moved inside the square bracket as well.

(and `propose` can be turned into a link to)